### PR TITLE
fix:modifying js code themes

### PR DESCRIPTION
**Pull request by ValeraS**
A possible solution for #227

### CODE REVIEW

**Comment by coolshaurya** [comment]
A screenshot highlighting the changes will greatly help the maintainers of freeCodeCamp understand the change and its value.

File: src/templates/Challenges/classic/Editor.js [hunk 1]
```diff
@@ -65,6 +65,29 @@ class Editor extends PureComponent {
     document.removeEventListener('keyup', this.focusEditor);
   }
 
+  editorWillMount(monaco) {
+    const yellowCollor = 'FFFF00';
```
Comment (Bouncey) [inline src/templates/Challenges/classic/Editor.js]: We can move these colour definitions outside of the class so they are not redefined for every editor instance

File: src/templates/Challenges/classic/Editor.js [hunk 2]
```diff
@@ -65,6 +65,29 @@ class Editor extends PureComponent {
...
+    const lightBlueColor = '9CDCFE';
+    const darkBlueColor = '00107E';
+    monaco.editor.defineTheme('vs-dark-custom', {
+      base: 'vs-dark',
+      inherit: true,
+      rules: [
```
Comment (Bouncey) [inline src/templates/Challenges/classic/Editor.js]: One of these rules has operands (`--`, `++`, `>=` etc...) the came colour as semi-colons (`;`). Could we change that?

Comment (ValeraS) [reply src/templates/Challenges/classic/Editor.js]: Default js tokenizer doesn't distinguish this symbols. They are all delimiters. We can separate them with custom tokenizer only.

File: src/templates/Challenges/classic/Editor.js (hunk 1 above)
Comment (ValeraS) [reply src/templates/Challenges/classic/Editor.js]: While we don't use monaco-editor-webpack-plugin #36, monaco appears in the global context after the first instantiating of MonacoEditor. In this situation, we can add a flag in the global context about we set themes already or take the monaco loader from the MonacoEditor and load the monaco by yourself.

By the way, editorWillMount executes only once for each lesson.

**Pull request merged**